### DISCUSSION TITLE
PD gains use different units in Isaac Lab and USD

### DISCUSSION THREAD

**Discussion by Wang-Haitong** [isaac-sim/IsaacLab#881] [General]
Hi, I recently found that the damping and stiffness coefficient in USD and Isaac Lab use different units. In Isaac Sim GUI, we can see that damping and stiffness uses **degree**; while in Isaac Lab, these two coefficients use **radian**. And when we set damping and stiffness to None in actuator cfg, Issac Lab would use the default values in USD prim, which would be multiplied by 180/PI. 

I think it would be better if this difference can be mentioned in the docstring of actuator cfg and documentation as it can cause problems that are hard to debug.

**Comment by Dhoeller19** [comment]
Hi @Wang-Haitong,
Indeed, it is a great idea to mention this in the docs.
